running.html

```diff
@@ -166,15 +166,14 @@
                 <li>Add the following line to <tt>/etc/apt/sources.list</tt>. Replace the distribution name as appropriate.
                   <pre>deb http://repo-cockpitproject.rhcloud.com/debian/ jessie main</pre></li>
                 <li>Import Cockpit's signing key to the apt sources keyring:
-                  <pre>sudo apt-key adv --keyserver sks-keyservers.net --recv-keys F1BAA57C</pre></li>
+                  <pre>sudo apt-key adv --keyserver sks-keyservers.net --recv-keys 0D2A45C3F1BAA57C</pre></li>
+                <li>Verify fingerprint:
+                  <pre>sudo apt-key finger F1BAA57C
+# compare output: Key fingerprint = FD9A 5764 17F7 B1D8 63C4  7A5A 0D2A 45C3 F1BA A57C</pre></li>
```

Comment:
These are commands to run, so the comment here looks a bit strange:
![screenshot from 2017-04-04 11-11-33](https://cloud.githubusercontent.com/assets/8711649/24649519/a4828ade-1927-11e7-9fbf-853872930b7b.png)
I recommend adding that text below the command to run, since it's nothing to copy & paste. The key to compare can look different / monospaced.